openedx/features/survey_report/models.py

```diff
@@ -35,3 +35,31 @@ class SurveyReport(models.Model):
     class Meta:
         ordering = ["-created_at"]
         get_latest_by = 'created_at'
+
+
+class SurveyReportUpload(models.Model):
+    """
+    This models stores information about send request of a survey report.
```

Comment:
I'd like this better instead of just _send request_
```suggestion
    This model stores the result of the POST request made to an external service after generating a survey report.
```